apps/src/code-studio/components/stageExtras/MazeThumbnail.jsx

```diff
@@ -5,7 +5,7 @@ import {getSubtypeForSkin} from '@cdo/apps/maze/mazeUtils';
 import MazeMap from '@cdo/apps/maze/mazeMap';
 import drawMap from '@cdo/apps/maze/drawMap';
 
-const assetUrl = path => '/blockly/' + path;
+const assetUrl = path => 'https://studio.code.org/blockly/' + path;
```

Comment:
That doesn't differentiate between local storybook and deployed storybook, but as an MVP I'm fine with all storybooks referencing production assets.  If you want to customize only the deployed storybook, we could probably add an environment variable to [this line](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/script/deploy-storybook.sh#L28) that got used in the storybook build config to set a different inlined constant.